lib/ansible/plugins/action/enos_config.py

```diff
@@ -0,0 +1,128 @@
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
```

Comment:
This should be GPL
Can you also please update `enos.py`

Reply:
Changing it to GPL for this file in this PR. The enos.py will be done in another PR

Reply:
Will do it in another PR